contrib/librespot.user.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=An open source Spotify client
+Documentation=https://github.com/librespot-org/librespot
+Documentation=https://github.com/librespot-org/librespot/wiki/Options
+
+[Service]
+Restart=always
+RestartSec=10
+ExecStart=/usr/bin/librespot --name "%u@%H"
```

Comment:
The format of the name argument is different to the other service. Why?

Reply:
Basically if i understand it correctly User@Host vs Librespot on Host.

Reply:
The other service can be run as `systemctl start something@librespot", and the service will make itself discoverable under the name `something@$HOSTNAME`.

I didn't see this adding much value for the user-managed service, TBH, and feel that `$USER@$HOSTNAME` is a bit more obvious.

Reply:
> systemctl start something@librespot

Can it? Doesn't this require a template unit file?  And even if you could, then it would advertise as "something on $HOSTNAME. Your new user service will advertise as "$USER@$HOSTNAME". They should just be the same style. "@" or "on". It's weirdly inconsistent.

Reply:
I've made these consistent.